### ISSUE TITLE
Support publication/update date in feeds

### ISSUE THREAD

**Issue by ArthurHoaro** [shaarli/Shaarli#563]
Currently, Shaarli only store the creation date of a shaare. So the dates are never updated in the feeds, which could make feed syndication services to ignore updated descriptions.

**Comment by ArthurHoaro** [comment]
Of course it's not supported by RSS.

**Comment by nodiscc** [comment]
AFAIK the only way to have RSS readers consider the item as updated is to change the item's `<pubDate>`. Shaarli does not store this information (update time). A workaround would be re-posting the shaare and deleting the previous occurence. This ("bump") feature was mentioned in https://github.com/shaarli/Shaarli/issues/186

**Comment by alexisju** [comment]
I personaly don't like the idea to repost edited share with a new date! 

Neither that the main rss/atom feed include old shares recently edited as new items. 
(I use 3rd party tool to reshare my shares from rss/atom feed ; in most situation i'll be very annoyed if this behavior was enabled by defaut).

However add a new function to display last edit date into templates (even rss/atom feed) could be very interesting.

**Comment by ArthurHoaro** [comment]
Good points @alexisju. I worked a bit on it yesterday, I went this way:

  * Store the edit date in the datastore (this will also make it usable in the templates).
  * Use `published`/`updated` tags in ATOM feed.
  * Ignore RSS feed, or use the ATOM extension:

```xml
<atom:updated>[date]</atom:updated>
```
But I'll probably just ignore RSS, because the extension isn't widely supported, and RSS/ATOM date formats aren't the same, which will make me do a ugly hack in `FeedBuilder`.
